docs/src/dataloader.md

```diff
@@ -168,3 +188,50 @@ public interface IUsersStore
 	Task<IEnumerable<User>> GetAllUsersAsync();
 }
 ```
+## Usage *without* Dependency Injection
+Because the DataLoaderContext is defined in the GraphQL Root object we can get it in our resolvers like this:
+```csharp
+public class ordertype : ObjectGraphType<order>
+{
+    public ordertype()
+    {
+        Name = "Ordertype";
+        UserStore store = new UserStore();
+
+        Field<IntGraphType>("id");
+        Field<StringGraphType>("userId");
+        Field<IntGraphType>("category");
+        Field<StringGraphType>("name");
+        Field<DateGraphType>("date");
+        Field<Usertype, User>.Name("user").ResolveAsync(ctx =>
+        {
+            store.SetResolveContext(ctx); //Make ctx availabele
+            var loader = ctx.RootValue.Context as DataLoaderContext.GetOrAddBatchLoader<int, User>("GetUserIds", store.GetUserIds);
+
+            return loader.LoadAsync(ctx.Source.userId);
+        });
+    }
+}
+
+```
+And we can define the UserStore like this:
+```csharp
+public class UserStore
+{
+    private object ctx;
+    public void SetResolveContext(object c)
+    {
+        ctx = c;
+    }
+    public async Task<Dictionary<int, User>> GetObjektByIdAsync(IEnumerable<int> userIds)
+    {
+        var users = db.Users.. //make db call
+        Dictionary<int, User> dic = new Dictionary<int, User>();
+        
+        foreach (var i in users)
+            dic.Add(userIds(users.IndexOf(i)), i);
```

Comment:
This code isn’t making much sense to me. I’m guessing this is a mix of C# and VB?  Can you update this to make a bit more sense?